src/utilities/intel/getLatLongByCity.js

```diff
@@ -0,0 +1,71 @@
+export default async function getLatLongByCityname(city, country) {
+  // Check if city has been set
+  if(city && country){
+    // Remove spaces from strings
+    city = city.trim();
+    country = country.trim();
+
+    // Set API link and infuse city and API key
+    const api = `https://api.opencagedata.com/geocode/v1/json?q=${city}
+&key=${process.env.REACT_APP_LATLONG_APIKEY}
```

Comment:
I think there are some indents missing. You should add them.

Reply:
Wrong. If indents are present, the url contains those indents.